### DISCUSSION TITLE
Palm Springs 54144 - PCBv2 Red LLC resistors question.

### DISCUSSION THREAD

**Discussion by gavinchan55** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#435] [Q&A]
Ive ordered PCBv2 and will be using the smaller RED LLC. Is it still recommended to remove all the resistors on that board and replace with wires as noted in the instructions? Or did the instructions mean to replace the resistors on Erics PCB with wires? :(

**Comment by visualapproach** [comment]
I think you are refferring to Eric's PCB. You don't need any bridges on the PCB V2 unless you want to run without LLC, but then you wouldn't replace any resistors. Now, cheer up man, what's up with that unhappy face?

**Reply by gavinchan55** [reply]
Yeah, sad face because I feel like an idiot, because I interpreted the instructions as 'replace the resistors on the RED LLC with wires' 😆  hahaha. I think I butchered it. Oh well, I got some practice with microsoldering at least. Thanks for the response 👍
